pegasus/sites.v3/code.org/public/css/mobile_header.css

```diff
@@ -0,0 +1,165 @@
+#pageheader-wrapper {
+  z-index: 5;
+}
+
+#nav-hamburger {
+  cursor: pointer;
+  padding: 10px 35px 16px 0px;
+  height: 20px;
+}
+
+#nav-hamburger span, #nav-hamburger span:before, #nav-hamburger span:after {
+  cursor: pointer;
+  border-radius: 1px;
+  height: 3px;
+  width: 25px;
+  border-radius: 2px;
+  background: white;
+  position: absolute;
+  display: block;
+  content: '';
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+
+#nav-hamburger span:before {
+  top: -10px;
+}
+
+#nav-hamburger span:after {
+  bottom: -10px;
+}
+
+#nav-hamburger span, #nav-hamburger span:before, #nav-hamburger span:after {
+  transition: all 500ms ease-in-out;
+}
+
+#nav-hamburger.active span {
+  background-color: transparent;
+}
+
+#nav-hamburger.active span:before, #nav-hamburger.active span:after {
+  top: 0;
+}
+
+#nav-hamburger.active span:before {
+  transform: rotate(45deg);
```

Comment:
Is this supported in all of our [supported browsers](https://support.code.org/hc/en-us/articles/202591743-What-kind-of-devices-browsers-and-operating-systems-are-supported-by-Code-org-)? If not, what does it look like when not supported?